src/pages/Auth/Login.jsx

```diff
@@ -20,15 +21,9 @@ const Login = () => {
   const loginUser = async (e, enteredEmail, password) => {
```

Comment:
I guess error handling is weak here.

Reply:
Form is taking care of it. i will add formik in the coming days after working on post functionality